examples/swap_networks.py

```diff
@@ -0,0 +1,83 @@
+"""Demonstrates swap networks."""
```

Comment:
So this demonstrates swap networks by generating a QAOA circuit as an example?  If so, is this another way of implementing https://github.com/quantumlib/Cirq/pull/1323/files?

@Strilanc  This uses `contrib` and i'm wondering if we should have rules about using `contrib` procedures in our examples.

Reply:
It also does more than QAOA. Apologies.

Reply:
It does both more and less than QAOA (i.e. no optimization, evaluation, etc.). The goal was to just show how to use the swap network code, and QAOA is a common, simple use case.

I'm fine with something like a "contrib examples" directory, though it doesn't seem necessary to me. (The examples are standalone files, so there's no risk of introducing indirect dependencies.)